pallets/msa/src/benchmarking.rs

```diff
@@ -156,6 +156,9 @@ benchmarks! {
 		assert_ok!(Msa::<T>::add_public_key_to_msa(RawOrigin::Signed(provider_public_key).into(), caller_and_delegator_public_key.clone(), owner_signature,  new_public_key_signature, add_key_payload));
 
 	}: _(RawOrigin::Signed(caller_and_delegator_public_key), new_public_key)
+	verify {
+		assert_eq!(frame_system::Pallet::<T>::events().len(), 1);
```

Comment:
Is this to verify if the extrinsic is successful?  If that's the case this should be fine for now but it will break if we add more events later. It seems the more correct thing to do is to assert the call, but that seems to be not compatible with the shorten macro.

Reply:
Yup. It is to check that we are not accidentally erroring out early and, as a result, not benchmarking what we want. I think this can be updated with a specific log message when we update.